opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -90,7 +90,7 @@ exports.init = function(options) {
             $("table[data-udf-id] .placeholder").css('display', 'none');
         },
 
-        getDataToSave = function() {
+        getDataToSave = options.getDataToSave || function() {
```

Comment:
@RickMohr, I _am_ using `options.onSaveBefore`, for the confirmer.

It doesn't appear to be intended for gathering data, since `getDataToSave` is called unconditionally, and will not work with the data for this page.

But I'll go look again.